test/Table_Tests/src/Util.enso

```diff
@@ -11,29 +11,14 @@ polyglot java import org.enso.base_test_helpers.FileSystemHelper
 Table.should_equal : Any -> Integer -> Any
 Table.should_equal self expected frames_to_skip=0 =
     loc = Meta.get_source_location 1+frames_to_skip
-    case expected of
-        _ : Table ->
-            tables_equal t0 t1 =
-                same_headers = (t0.columns.map .name) == (t1.columns.map .name)
-                same_columns = (t0.columns.map .to_vector) == (t1.columns.map .to_vector)
-                same_headers && same_columns
-            equal = tables_equal self expected
-            if equal.not then
-                msg = 'Tables differ at '+loc+'.\nActual:\n' + self.display + '\nExpected:\n' + expected.display
-                Test.fail msg
-        _ -> Test.fail "Got a Table, but expected a "+expected.to_display_text+' (at '+loc+').'
+    Panic.catch Any (self.should_equal_impl expected loc) error->
+        Test.fail error.payload 
 
 Column.should_equal : Any -> Integer -> Any
 Column.should_equal self expected frames_to_skip=0 =
     loc = Meta.get_source_location 1+frames_to_skip
-    case expected of
-        _ : Column ->
-            if self.name != expected.name then
-                Test.fail "Expected column name "+expected.name+", but got "+self.name+" (at "+loc+")."
-            if self.length != expected.length then
-                Test.fail "Expected column length "+expected.length.to_text+", but got "+self.length.to_text+" (at "+loc+")."
-            self.to_vector.should_equal expected.to_vector 2+frames_to_skip
-        _ -> Test.fail "Got a Column, but expected a "+expected.to_display_text+' (at '+loc+').'
+    Panic.catch Any (self.should_equal_impl expected loc) error->
```

Comment:
I'm not a fan of throwing raw `Text`s. Ideally I think we should create a `type` wrapping it.

Otherwise, I'd at least be catching `Text` here, not just `Any`. If you catch `Any`thing other than `Text`, you'll get a type error in `Test.fail` afterwards.

Reply:
Fixed